app/main.py

```diff
@@ -2,0 +1,33 @@
+class Animal:
+    alive = []
+
+    def __init__(
+            self, name: str,
+            health: int = 100,
+            hidden: bool = False
+    ) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        self.__class__.alive.append(self)
```

Comment:
Appending `self` to `self.__class__.alive` is correct for adding the current instance to the list of living animals. Ensure that this behavior is desired for all subclasses of `Animal` as well.